.github/workflows/all-green.yml

```diff
@@ -0,0 +1,21 @@
+name: Check Pull Request CI Status
+
+on:
+  pull_request:
+    types:
+      - opened
+      - synchronize
+      - reopened
+
+permissions:
+  checks: read
+  statuses: read
+
+jobs:
+  all-jobs-are-green:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Run Ensure CI Success
+        uses: DataDog/ensure-ci-success@v1
```

Comment:
We should pin against the specific commit hash for this branch/tag.

Reply:
done in [7a6df8f](https://github.com/DataDog/dd-trace-go/pull/3487/commits/7a6df8f8bfd23311e3656bce6cc04b8f77bc3ecd)